pages/idt.adoc

```diff
@@ -0,0 +1,171 @@
+---
+title: Interrupt descriptor table
+tags: assembly, x86, x64, sysv, msvc
+category: Interrupts
+description: IDT spec and tutorial
+---
+:source-language: c
+
+== Interrupt Descriptor Table
+
+// TODO: articles "GDT" and "Interrupts" are not written yet
+
+Interrupt Descriptor Table is a binary data structure specific to IA-32 (xref:x86[x86]) and x86_64 architectures.
+It is used by the CPU to lookup an Interrupt Service Routine (ISR) when an xref:interrupts[Interrupt] is issued.
+It is Protected Mode and Long Mode counterpart to Real Mode Interrupt Vector Table (IVT).
```

Comment:
"protected mode" and "long mode" are apparently lower case
"interrupt" and "interrupt service routine" are definitely lower case